### PR TITLE
Zodiac developments

### PR DESCRIPTION
**Pull request by ohuynh21**
The last merges kind of broke the zodiac wheels. I think the issue has to do with the rounding of the angles for each wheel if you rotate it beyond 360 and -360 degrees. When that happens the wheel is always rotated back to something in the range 0-360 which causes it to rotate quite dramatically especially if the pre rounded rotation angle is something large like 720 for instance. This probably has something to do with the introduction of the new mod function. 

In addition it seems that the zodiac mapping/date ranges no longer line up.

I've rolled back the functions to their last stable versions and they seem to work just fine now.

Some before and after videos have been linked below for visual clarity and understanding. Note the console output and behavior of the wheels.

[Before](https://drive.google.com/file/d/1iO920f1FRzE3CGF06HaAdccYG1Ss2-PF/view?usp=sharing)

[After](https://drive.google.com/file/d/1FjZOZo3WiNthyVDBwLXGuAkvOKaAlPr0/view?usp=sharing)

I've also readjusted the size of some of the buttons and made the arrows slightly smaller.

### CODE REVIEW

File: source/PalmReading/script.js
Comment (SheepTester) [inline source/PalmReading/script.js:1]: thanks for formatting palm reader as well

**Review by SheepTester — APPROVED** [review]
looks good, we probably would want to write some tests for these some day

File: source/PalmReading/script.js
Comment (ohuynh21) [reply source/PalmReading/script.js:1]: yup i'll write some unit tests for the DOM-less functions you separated tomorrow morning/afternoon

**Comment by ohuynh21** [comment]
#23 added some sliding animations to the zodiac wheels to allow them to horizontally slide onto the page when the page loads. Also made the buttons and arrows appear after some delay. idk if you want to keep these, but I figured it was better than just having two massive wheels load immediately when you visit the page.

**Pull request merged**